tangent/models/project_task.py

```diff
@@ -0,0 +1,31 @@
+from odoo import api, models, fields
+import random
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    invoice_number = fields.Many2one(readonly=True, comodel_name="account.move", compute="_generate_invoice")
```

Comment:
just a heads up by default computed fields are readonly so you don't need to add readonly